### ISSUE TITLE
component page breadcrumbs should have an aria-label

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/pulfalight#1778]
refs #1717 
[full list of issues for audit1717](https://github.com/pulibrary/pulfalight/issues?q=is%3Aissue%20state%3Aopen%20audit1717)

### Location of problem on site
component page, e.g. https://findingaids.princeton.edu/catalog/C0108_c3741

### Description of error / warning

When navigating with VoiceOver, it starts to read the breadcrumbs area just like link after link. This section should have an aria-label to give more context

### WCAG success criterion

[4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG22/#name-role-value)

"ARIA is used appropriately to enhance accessibility when HTML is not sufficient."

### WCAG conformance level
A

### Recommended fix
Add an aria label to the div around the breadcrumbs

### Tool used to find the problem
VoiceOver in safari

### Screenshot
n/a
